packages/plugin-compiler/src/main/kotlin/io/realm/compiler/Identifiers.kt

```diff
@@ -26,6 +26,7 @@ internal object Names {
     val REALM_OBJECT_COMPANION_PRIMARY_KEY_MEMBER: Name = Name.identifier("${REALM_SYNTHETIC_PROPERTY_PREFIX}primaryKey")
     val REALM_OBJECT_COMPANION_SCHEMA_METHOD: Name = Name.identifier("${REALM_SYNTHETIC_PROPERTY_PREFIX}schema")
     val REALM_OBJECT_COMPANION_NEW_INSTANCE_METHOD = Name.identifier("${REALM_SYNTHETIC_PROPERTY_PREFIX}newInstance")
+    val REALM_OBJECT_COMPANION_EXCLUDED_PROPERTIES_MEMBER: Name = Name.identifier("${REALM_SYNTHETIC_PROPERTY_PREFIX}excludedProperties")
```

Comment:
I don't really see why we need a list of excluded properties!?

Reply:
It facilitates accessing the excluded properties from a test case, avoiding writing ugly constants, and it can help to debug.

Reply:
If you are referring to https://github.com/realm/realm-kotlin/pull/363/files#diff-899d275cf0f244a072efa97142af33cf9483a22d0fc6f51a0cf45514533f53d4R184, then I guess you could just assert
`$realm$fields.count` instead.